### PR TITLE
Separation of Add-On-related configs from CMake Tools configs

### PR DESCRIPTION
**Pull request by bghszabi**
**Update Tools submodule before merging!**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,7 +8,9 @@ set_property (GLOBAL PROPERTY USE_FOLDERS ON)
 
 set (CMAKE_CONFIGURATION_TYPES Debug;Release;RelWithDebInfo)
 set (AC_API_DEVKIT_DIR ${API_DEVKIT_DIR} CACHE PATH "API DevKit directory.")
-set (AC_ADDON_NAME "ExampleAddOn" CACHE STRING "Add-On name.")
+if (NOT DEFINED AC_ADDON_NAME)
```
Comment (kovacsv) [inline CMakeLists.txt:11]: I guess it can be a cache variable like the others:
```
set (AC_ADDON_NAME "ExampleAddOn" CACHE STRING "Add-On name.")
```

**Review by kovacsv — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (imecsBulcsu) [reply CMakeLists.txt]: We do not need to set it in the CMakeLists file, we decided that AC_ADDON_NAME is a mandatory argument for generating the solution. This set and the if should get removed according to the latest Tools repository changes, and the readme should be modified accordingly.

Comment (mKrisztianGraphi) [reply CMakeLists.txt]: Fixed

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -31,4 +31,4 @@ jobs:
       
       - name: Run build script
         run: |
-          python Tools/BuildAddOn.py --configFile config.json --acVersion ${{ matrix.ac-version }}
+          python Tools/BuildAddOn.py -c config.json --acVersion ${{ matrix.ac-version }}
```
Comment (esfarkas) [inline .github/workflows/build.yml:34]: lets write out the -c as -configFile again, easier to understand for new developers

**Pull request merged**